npm/ng-packs/packages/schematics/package.json

```diff
@@ -9,20 +9,20 @@
   "license": "MIT",
   "schematics": "./collection.json",
   "dependencies": {
-    "@angular-devkit/core": "~15.0.1",
-    "@angular-devkit/schematics": "~15.0.1",
-    "@angular/cli": "~15.0.1",
+    "@angular-devkit/core": "16.0.1",
+    "@angular-devkit/schematics": "16.0.1",
+    "@angular/cli": "16.0.1",
```

Comment:
*devkits* are fine but, @angular/cli package may can effect npm install after create project with CLI like this line https://github.com/abpframework/abp/blob/cce33573251553f817e7e88cafd5bd7f8285caf1/npm/ng-packs/packages/account-core/package.json#L13

Maybe not now but for the future versions it should not be conflict